modules/raster-layers/package.json

```diff
@@ -32,11 +32,11 @@
     "lodash.isequal": "^4.5.0"
   },
   "peerDependencies": {
-    "@deck.gl/core": "^9.1.12",
-    "@deck.gl/layers": "^8.8.0",
-    "@deck.gl/mesh-layers": "^8.8.0",
-    "@luma.gl/core": "^8.2.0",
-    "@luma.gl/engine": "^8.2.0"
+    "@deck.gl/core": "~9.1.12",
+    "@deck.gl/layers": "~8.8.0",
```

Comment:
Is this module intended to be using both v8 and v9?

Reply:
That seems more like an indicator that the module isn't being maintained by anyone. I wouldn't expect these major versions to be compatible